pyproject.toml

```diff
@@ -104,8 +105,20 @@ artifacts = ["altair/jupyter/js/index.js"]
 
 [tool.hatch.envs.default]
 features = ["all", "dev"]
+# https://hatch.pypa.io/latest/how-to/environment/select-installer/#enabling-uv
+installer = "uv"
 
 [tool.hatch.envs.default.scripts]
+generate-schema-wrapper = [
```

Comment:
TODO
----
Adapt this into `generate_schema_wrapper.py` so the test can pass CI.

Reply:
See:
- https://github.com/vega/altair/pull/3431#issuecomment-2160849568
- [`9582971` (#3431)](https://github.com/vega/altair/pull/3431/commits/958297128bb3912ad5b730477b9fd48d380dfcd1)